### ISSUE TITLE
Setting FingerPrint

### ISSUE THREAD

**Issue by ngjermundshaug** [libragnar/ragnar#101]
I am setting fingerprint like this:

    var fp = new Fingerprint("LT", 1, 0, 6, 0);
    libTorrentServer = new Session(fp);

But uTorrent and other clients are getting some kind of rubbish as the PeerID:

![image](https://cloud.githubusercontent.com/assets/1677884/9229138/ede58d74-411b-11e5-8c41-6a92e25871cb.png)

Anyone else getting the same problem?

Also - upon initial seeding - Clients are only getting approx 10 kbps. After starting/stopping other clients several times - they get 2-3 megabyte/sec. I would like to get max speed from the beginning. Is it possible?

**Comment by ngjermundshaug** [comment]
I also tried with a Ragnar.dll that I compiled myself - getting same problem - Client name is all obfuscated.

![image](https://cloud.githubusercontent.com/assets/1677884/9258940/8e017db2-41fe-11e5-80de-6ea1f4dde43b.png)

**Comment by ngjermundshaug** [comment]
Setting AnonymousMode to false fixed the issue.

**Comment by MozzieMD** [comment]
How to set AnonymousMode to false?

**Comment by vktr** [comment]
@MozzieMD It is set to false by default.

**Comment by ngjermundshaug** [comment]
@vktr I don't think it is set to false by default. I had to set it to false myself in order to resolve the issue.

    var settings = libTorrentServer.QuerySettings();
    settings.AnonymousMode = false;
    libTorrentServer.SetSettings(settings);
